pkg/domain/infra/runtime_libpod.go

```diff
@@ -123,7 +123,7 @@ func getRuntime(ctx context.Context, fs *flag.FlagSet, opts *engineOpts) (*libpo
 	storageOpts := types.StoreOptions{}
 	cfg := opts.config
 
-	storageSet := false
+	storageSet := cfg.ContainersConfDefaultsRO.Engine.StaticDir != ""
```

Comment:
This part here is a bit disturbing.  As mentioned in the commit message.  Massaging the various storage directories is scattered over many places, and I would expect such defaults to be set in `libpod` only.  But I didn't investigate further.

Reply:
Still some farts in the code.